packages/lists/package.json

```diff
@@ -20,7 +20,6 @@
     "code-style": "just-scripts code-style",
     "start": "just-scripts dev",
     "start-test": "just-scripts jest-watch",
-    "update-api": "just-scripts update-api",
```

Comment:
This script and associated output file will have to get reintroduced once @soettl introduces downstream List improvements. The blocker _was_ Fabric 7 upgrade, but we've recently migrated and should be unblocked. I'd keep it.

Reply:
The task isn't actually hooked up to do anything at the moment (api-extractor doesn't run as part of lists build), so I thought it would make more sense to get rid of it for now and re-introduce properly if/when it's actually needed.

Reply:
Basically make it clear that api-extractor is **not** running for this project until Sebastian is ready to set it up, rather than having some elements (the api.md and script) which make it appear to be running when it's actually not.